generator/whitelist.ts

```diff
@@ -318,6 +318,10 @@ const whitelist: WhitelistConfig[] = [
         basePath: 'storagesync/resource-manager',
         namespace: 'Microsoft.StorageSync',
     },
+    {
+        basePath: 'storSimple1200Series/resource-manager',
+        namespace: 'Microsoft.StorSimple',
+    },
```

Comment:
Because there are two separate directories for the same RP (storsimple8000series and storsimple1200series), please could you add a suffix here to avoid the files overwriting each other:
`suffix: '1200',`